AppCenter/AppCenter/Internals/Channel/MSChannelGroupDefaultPrivate.h

```diff
@@ -5,7 +5,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @class MSAppCenterIngestion;
 
-@interface MSChannelGroupDefault () <MSAuthTokenContextDelegate, MSChannelDelegate>
```

Comment:
Is `MSAuthTokenContextDelegate` still being used?

Reply:
@jaelim-ms yes, at least in `feature/storage` branch.